src/ui/pages/SwapForm/SwapForm.tsx

```diff
@@ -139,9 +143,9 @@ function FormHint({
     : null;
   const exceedsBalance = Number(spendInput) > Number(positionBalanceCommon);
 
-  const showPriceImpactWarning =
-    priceImpact?.kind === 'loss' &&
-    (priceImpact.level === 'medium' || priceImpact.level === 'high');
+  const priceImpactWarningVisible = priceImpact
```

Comment:
Is it `Boolean(priceImpact && isSignificantValueLoss(priceImpact))`?

Reply:
ah, yes :)